models/migrations/v62.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+	"github.com/go-xorm/xorm"
+)
+
+func addLanguageSetting(x *xorm.Engine) error {
+	type User struct {
+		Language string `xorm:"VARCHAR(5)"`
+	}
+
+	if err := x.Sync2(new(User)); err != nil {
```

Comment:
Is it possible that if a user logs in after an upgrade of the database, the language he/she is currently using is set as `language` in database? Do you think that the default language should be `en_US` (at database migration)?

Reply:
I haven't thought of that, I'll implement it.

Reply:
Done.